data/usegalaxy.eu.yaml

```diff
@@ -0,0 +1,39 @@
+title: European Galaxy Server
+provider: European Galaxy Project, NFDI4Plants, NFDI4BioImaging, EOSC
+service_url: https://live.usegalaxy.eu
+support: contact@usegalaxy.eu  
+health_api_url: https://status.galaxyproject.org
+documentation_url: https://training.galaxyproject.org
+target_group_open_for: Open for everyone
+login_process: Login via OpenIDC (LS-AAI, DataPLANT)
+features:
+  version: Galaxy 23.0
```

Comment:
`version: Galaxy 23.0`

Here should be the JupyterHub or BinderHub version, and the JupyterLab version.

Reply:
I added v3+, thanks!